src/components/categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Author: Gabriel Rozas</div>
+);
```

Comment:
- [ ] [OPTIONAL] I see that there has been some confusion due to this instruction: 
`IMPORTANT in the design you can see an input for a category - please replace it with an author.`
Please kindly note that it is referring to the category input in the form and not the category page (component). Kindly update your code so that the navbar should contain `Books` and `Categories` as shown in the design to avoid confusion later on. 

![Screenshot from 2023-06-09 15-08-12](https://github.com/EmiLoKiJu/bookstore/assets/65233860/d994b7c5-6df9-4e34-bc17-abe590a434c8)


![Screenshot from 2023-06-09 15-09-41](https://github.com/EmiLoKiJu/bookstore/assets/65233860/51a4a9b6-0c36-435c-956a-e72a9c20dd47)